src/main/java/com/google/sps/server/HandleSchedules.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.sps.server;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.Iterator;
+
+import com.google.gson.Gson;
+import com.google.sps.data.*;
+
+/**
+ * Exports the ordered schedules to secondary calendars on the user's Google calendar    
+ */
+@WebServlet("/handleSchedules")
+public class HandleSchedules extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      com.google.api.services.calendar.Calendar client = Utils.loadCalendarClient();
```

Comment:
Add `com.google.api.services.calendar.Calendar` to your imports since it's the only `Calendar` class in this file.